.github/pull_request_template.md

```diff
@@ -0,0 +1,24 @@
+# Describe what this PR does #
+
+Provide some context for the reviewer
+
+## Is there anything that requires special attention ##
+
+Do you have any questions? Did you do something clever?
+
+Provide any external context for the change, if any.
+
+For example: Kubernetes links that explain why a change is
+required, or CSI spec related changes that may
+need to be pointed to, or generic golang related practices
+that may need to be pointed to.
+
+## Related issues ##
+
+Mention any github issues relevant to this PR
+
+## Future concerns ##
+
+list items that are not part of the PR and do not impact it's
```

Comment:
(minor nit) Capitalize first letter of this sentence

(nit) ending should be "...but are work items that can be taken up ~~in the~~ subsequently."

Reply:
Done.